release-notes/unreleased/IGAPP-678-add-hungarian.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-678
+show_in_stores: false
+platforms:
+  - android
+  - ios
+en: Added support for hungarian language.
```

Comment:
I think it is rather confusing since this is just about content language and not about ui language. So if a user sees this note in the store and it is not available in his city he might be very confused.